chromium_src/chrome/browser/ui/webui/settings/people_handler.cc

```diff
@@ -9,7 +9,10 @@
   sync_status->SetBoolean(                \
       "firstSetupInProgress",             \
       service && !disallowed_by_policy && \
-          !service->GetUserSettings()->IsFirstSetupComplete());
+          !service->GetUserSettings()->IsFirstSetupComplete()); \
+  sync_status->SetBoolean(                \
+      "isEngineInitialized",             \
```

Comment:
We absolutely do not need this, `PeopleHandler::PushSyncPrefs` will check `IsEngineInitialized` before firing the event, that is why we should change encryption/decryption handling in `handleSyncStatus_`. We still need `handleSyncPrefsChanged_`